tests/next_tests/unit_tests/iterator_tests/test_pretty_parser.py

```diff
@@ -195,6 +202,17 @@ def test_stencil_closure():
     assert actual == expected
 
 
+def test_set_at():
+    testee = "y @ cartesian_domain() ← x;"
```

Comment:
```suggestion
    testee = "y ← x @ cartesian_domain();"
```
I prefer the existing format for a closure.

Reply:
The change is intentional and expresses that the result of the applied field operation is written `@ domain`. While the stencil_closure semantic was executed over a domain. Alternative syntax could be `y ← x [cartesian_domain()];` with `[]` getitem semantics, but I would prefer my current proposal.

Reply:
From a conceptual point I fully I agree and the meaning of the new syntax is easier to understand without prior knowledge, however when working with the IR one is rarely interested in the domain, but much more in the stencil itself. I'm ok with the new one, but don't be surprised about some complaints from me in the future :-)